src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/SasTokenControllerTest.java

```diff
@@ -30,23 +31,25 @@
     private MockMvc mockMvc;
 
     @Test
-    public void should_return_sas_token_when_service_configuration_is_available() throws Exception {
+    public void should_return_sas_token_for_sscs_when_service_configuration_is_available() throws Exception {
         String tokenResponse = this.mockMvc.perform(get("/token/sscs"))
             .andDo(print())
             .andExpect(status().isOk())
             .andReturn()
             .getResponse().getContentAsString();
 
-        final ObjectNode node = new ObjectMapper().readValue(tokenResponse, ObjectNode.class);
-
-        Map<String, String[]> queryParams = PathUtility.parseQueryString(node.get("sas_token").asText());
+        verifySasTokenProperties(tokenResponse);
+    }
 
-        String currentDate = new SimpleDateFormat("yyyy-MM-dd").format(new Date());
+    @Test
+    public void should_return_sas_token_for_probate_when_service_configuration_is_available() throws Exception {
+        String tokenResponse = this.mockMvc.perform(get("/token/probate"))
```

Comment:
looks like it has become parameterised test now. even thought only 2 tokens are tested here.